.devcontainer/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.222.0/containers/go/.devcontainer/base.Dockerfile
```

Comment:
This is rad! Can we add it to a separate PR?

Reply:
Sure thing, I've reverted my merge and I'll open a separate PR.

Reply:
https://github.com/tidbyt/community/pull/174